docs/core/tools/dotnet-nuget-trust.md

```diff
@@ -0,0 +1,193 @@
+---
+title: dotnet nuget trust command
+description: The dotnet nuget trust command gets or sets trusted signers to the NuGet configuration.
+author: erdembayar
+ms.date: 06/02/2021
+---
+# dotnet nuget trust
+
+**This article applies to:** ✔️ .NET 5.0.300 SDK, 6.0.100-preview.4 SDK and later versions
+
+## Name
+
+`dotnet nuget trust` - Gets or sets trusted signers to the NuGet configuration.
+
+## Synopsis
```

Comment:
The best model to follow for this article is the article for [dotnet sln](https://docs.microsoft.com/dotnet/core/tools/dotnet-sln) with the exception that we should adopt the way you show the available commands here in the main Synopsis section.